### PR TITLE
[fuchsia] Add BUILD.gn for flutter_localizations

### PR DESCRIPTION
**Pull request by kpozin**

### CODE REVIEW

File: packages/flutter_localizations/BUILD.gn [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+dart_library("flutter_localizations") {
+  package_name = "flutter_localizations"
+
+  # Can be left empty as analysis is disabled.
+  sources = []
+
+  disable_analysis = true
+
+  deps = [
```
Comment (jonahwilliams) [inline packages/flutter_localizations/BUILD.gn]: This needs a dependency on `intl`

**Comment by jonahwilliams** [comment]
+ @zanderso , does intl already get pulled into third_party or does this require a manual action during the roll?

**Comment by kpozin** [comment]
> does intl already get pulled into third_party or does this require a manual action during the roll?

FWIW, the [`flutter` GN target](https://github.com/flutter/flutter/blob/master/packages/flutter/BUILD.gn#L16) already depends on `"//third_party/dart/third_party/pkg/intl"`.

**Comment by zanderso** [comment]
Yes, intl is already pulled into the tree. This change lgtm.

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Comment by Hixie** [comment]
@kpozin for future PRs, please include a detailed description explaining what is happening. That way people who are doing archeology later can get the context required to determine what the PR was about. Thanks!

@jonahwilliams when reviewing PRs, please make sure PRs have descriptions. Thanks!

**Pull request merged**